core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -1340,6 +1340,10 @@ public boolean isKylinMetricsReporterForJobEnabled() {
         return Boolean.parseBoolean(getOptional("kylin.core.metrics.reporter-job-enabled", "false"));
     }
 
+    public String getKylinMetricsPrefix() {
+        return getOptional("kylin.core.metrics.prefix", "KYLIN").toUpperCase();
```

Comment:
Suggest to change config prefix "kylin.core.metrics" to "kylin.metrics", including other metrics props in last pr.

Reply:
Will change them all after merging cube planner phase two